assets/src/editor/blocks/home-page-hero/edit.js

```diff
@@ -70,7 +70,6 @@ const HomePageHeroBlock = ( { attributes, setAttributes, isSelected, setHeadingC
 				) }>
 					<ImagePicker
 						className="hero-home__image"
-						id={ imageId }
```

Comment:
Does this attribute need to be removed here? Seems like it might be useful to continue storing the id attribute, even if it's not output in the save component.

Reply:
I removed it to prevent the unmatched output error, but I did not fully tested this since I was not able to setup my local env.